docs/people/README.md

```diff
@@ -0,0 +1,12 @@
+# Opt-in Directory of People
+This is an *opt-in* directory of people involved in KYOSS in some manner.
+
+Adding your own name and details (opting-in) to this directory is done *by you* using the open-source behavior pattern faciliated by git.
+
+---
+
+
+| Name | e-mail | more |
+| --- | ---| --- |
+| Dave Campbell | <daveccampbell@gmail.com> | [personal member page](./davecampbell/)
```

Comment:
Do you really want to list your email address on a public / open web site like this (i.e., where it can be easily scraped)

You might want to link to your LinkedIn profile, or Github profile, or some other public way to contact you.  Just a suggestion...

Reply:
yeah, good idea to not make it that easy to harvest emails.
left just one additional column in the table - people can put what in whatever they choose to there to get more details.